docs/diga_api.md

```diff
@@ -0,0 +1,216 @@
+| 102          | 77AAAAAAAAAAADGE | Fresichaltcode nicht gefunden | Fachlicher Fehler. Der Fehler wird ausgegeben wenn bspw. die Zuordnung des Freischaltcodes zur DiGA nicht stimmt.                                                                      |
+| 200          | 77AAAAAAAAAAAGIS | Anfrage oder Datei ungültig   | Die Anfrage oder die Datei konnte nicht verarbeitet werden. Der Fehler wird bspw. bei einer Schemaverletzung ausgegeben.                                                               |
+| 201          | 77AAAAAAAAAAAGJC | Serverfehler                  | Technischer Fehler. Der Fehler wird bspw. bei einem Übertragungsfehler ausgegeben.                                                                                                     |
+| 202          | 77AAAAAAAAAAAGKD | Speicherfehler                | Technischer Fehler. Der Fehler wird bspw. bei einem Datenbankfehler ausgegeben.                                                                                                        |
+
+### Mapping file for insurances
+
+Data for one insurance is listed below.
+
+```xml
+<n1:Krankenkasseninformation Nummer="ID1">
+        <bas:Kostentraegerkuerzel>CH</bas:Kostentraegerkuerzel>
+        <bas:Kostentraegerkennung>109034270</bas:Kostentraegerkennung>
+        <bas:Name_des_Kostentraegers>BMW BKK   </bas:Name_des_Kostentraegers>
+        <bas:IK_des_Rechnungsempfaengers>109034270</bas:IK_des_Rechnungsempfaengers>
+        <bas:IK_Abrechnungsstelle>660500345</bas:IK_Abrechnungsstelle>
+        <bas:Name_Kommunikationsstelle>DIGA-BITMARCK</bas:Name_Kommunikationsstelle>
+        <bas:Endpunkt_Kommunikationsstelle>diga.bitmarck-daten.de</bas:Endpunkt_Kommunikationsstelle>
+        <bas:Versandart>1</bas:Versandart>
+        <bas:Postalische_Zusaetze>BMW BKK</bas:Postalische_Zusaetze>
+        <bas:Strasse_Postfach>Mengkofener Str.</bas:Strasse_Postfach>
+        <bas:Hausnummer_Postfachnummer>6</bas:Hausnummer_Postfachnummer>
+        <bas:PLZ>84130</bas:PLZ>
+        <bas:Ort>Dingolfing</bas:Ort>
+        <bas:Kontaktdaten_Technisch_Telefon>0800-24862725</bas:Kontaktdaten_Technisch_Telefon>
+        <bas:Kontaktdaten_Technisch_EMail>servicedesk@bitmarck.de</bas:Kontaktdaten_Technisch_EMail>
+</n1:Krankenkasseninformation>
+```
+
+The file can be updated by insurances which requires keeping the mapping file used for the client up-to-date. The field `Kostentraegerkuerzel` is what can be used to find insurance information based on the prescription code. The insurance information lists a field `Endpunkt_Kommunikationsstelle`. This is the base url for the endpoint for a specific insurance. There is no central diga endpoint for verifying and reimbursing DiGA apps and handling might differ among insurances. Some apis might also not support billing as of now. Invoices in these cases must be sent by email or post.
```

Comment:
Maybe mention that it can be updated quarterly with every new round of DiGA application deadlines

`diga/DiGA` to keep the formatting consistent

On the READ it mentions that the response in case billing is not supported provides information about how to proceed. https://github.com/alex-therapeutics/diga-api-client#project-status

Again the comment about linking to the Anlage header for the OpenAPI specification